From 8d86855b51c5ff4a159f6b5b11e4e438283ea3a0 Mon Sep 17 00:00:00 2001
From: Sandy Carter <bwrsandman@gmail.com>
Date: Sun, 30 Jan 2022 18:43:31 -0500
Subject: [PATCH 2/2] examples: Don't build on android by default

---
 cmake/examples.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/examples.cmake b/cmake/examples.cmake
index a6b8713..737b968 100755
--- a/cmake/examples.cmake
+++ b/cmake/examples.cmake
@@ -171,7 +171,7 @@ function( add_example ARG_NAME )
 	target_compile_definitions( example-${ARG_NAME} PRIVATE "-D_CRT_SECURE_NO_WARNINGS" "-D__STDC_FORMAT_MACROS" "-DENTRY_CONFIG_IMPLEMENT_MAIN=1" )
 
 	# Configure shaders
-	if( NOT ARG_COMMON AND NOT IOS AND NOT EMSCRIPTEN)
+	if( NOT ARG_COMMON AND NOT IOS AND NOT EMSCRIPTEN AND NOT ANDROID)
 		foreach( SHADER ${SHADERS} )
 			add_bgfx_shader( ${SHADER} ${ARG_NAME} )
 		endforeach()
-- 
2.35.1

